README.md

```diff
@@ -193,7 +195,7 @@ _TODO: realistic example of when you'd use this._
 
 Unlike the existing history API's `history.go()` method, which navigates by offset, navigating by key allows the application to not care about intermediate history entries; it just specifies its desired destination entry.
 
-There are also convenience methods, `appHistory.back()` and `appHistory.forward()`.
+There are also convenience methods, `appHistory.back()` and `appHistory.forward()`. Note that these only navigate through the _app history_ list, not the _joint session history_; that is, they do not navigate frames, and they cannot go back or forward to a cross-origin destination.
```

Comment:
Is that possible? What if:

1. PageA contains an iframe to PageB.
1. `appHistory.pushNewEntry(…)` in top level page.
1. Navigate iframe PageB to PageC.

Now we have a joint history with 3 items, and we're currently on the third of those items.

What does `appHistory.entries` look like? What happens if `appHistory.back()` is called? And assuming the first entry in `appHistory.entries` points to step 1 above, what happens if it's traversed to? Does it also navigate the iframe?

Reply:
Yeah, OK, I worked through this and it looks like you can navigate child frames. I don't think child frames can use app history to navigate parent frames...

I'll push a commit showing what I ended up with.